simplesamlphp/metadata/saml20-idp-remote.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
```

Comment:
Can we put this file in the secrets/not source controlled location? Then have a generic PHP script that pulls in the correct file?

Reply:
Done.